### PR TITLE
FIX: ALT attributes

### PR DESCRIPTION
**Pull request by estelle**
Udated a few alt attributes to make them useful

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/games/techniques/3d_collision_detection/index.md [hunk 1]
```diff
@@ -16,11 +16,11 @@ This article provides an introduction to the different bounding volume technique
 
 As with 2D collision detection, **axis-aligned bounding boxes** (AABB) are the quickest algorithm to determine whether the two game entities are overlapping or not. This consists of wrapping game entities in a non-rotated (thus axis-aligned) box and checking the positions of these boxes in the 3D coordinate space to see if they are overlapping.
 
-![](screen_shot_2015-10-16_at_15.11.21.png)
+![Two 3-D non-square objects floating in space encompassed by virtual rectangular boxes. ](screen_shot_2015-10-16_at_15.11.21.png)
```
Comment (teoli2003) [inline files/en-us/games/techniques/3d_collision_detection/index.md:19]:
```suggestion
![Two 3-D non-square objects floating in space encompassed by virtual rectangular boxes.](screen_shot_2015-10-16_at_15.11.21.png)
```

File: files/en-us/games/techniques/3d_collision_detection/index.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ function isPointInsideAABB(point, box) {
 
 Checking whether an AABB intersects another AABB is similar to the point test. We just need to do one test per axis, using the boxes' boundaries. The diagram below shows the test we'd perform over the X-axis — basically, do the ranges _A<sub>minX</sub>_–_A<sub>maxX</sub>_ and _B<sub>minX</sub>_–_B<sub>maxX</sub>_ overlap?
 
-![updated version](aabb_test.png)
+![Hand drawing of two rectangles showing the upper right cordern of A overlapping the bottom left corner of B, as the A-Xmax is greater than B-Xmin. ](aabb_test.png)
```
Comment (teoli2003) [inline files/en-us/games/techniques/3d_collision_detection/index.md:48]:
```suggestion
![Hand drawing of two rectangles showing the upper right corner of A overlapping the bottom left corner of B, as A's largest x coordinate is greater than B's smallest x coordinate.](aabb_test.png)
```

File: files/en-us/games/techniques/3d_collision_detection/index.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ Using bounding spheres to detect collisions is a bit more complex than AABB, but
 
 To check whether a sphere contains a point we need to calculate the distance between the point and the sphere's center. If this distance is smaller than or equal to the radius of the sphere, the point is inside the sphere.
 
-![](point_vs_sphere.png)
+![Hand drawing of a sphere with a point. The point is outside of the sphere, to the lower right of it. The distances is denoted by a dashed line from the center of the circle to the dot labeled D. A lighter line shows the radius, going from the center of the circle to the border of the circle, labeled R.](point_vs_sphere.png)
```
Comment (teoli2003) [inline files/en-us/games/techniques/3d_collision_detection/index.md:73]:
```suggestion
![Hand drawing of a 2D projection of a sphere and a point in a Cartesian coordinate system. The point is outside of the circle, to the lower right of it. The distance is denoted by a dashed line, labeled D, from the circle's center to the point. A lighter line shows the radius, labeled R, going from the center of the circle to the border of the circle.](point_vs_sphere.png)
```

File: files/en-us/games/techniques/3d_collision_detection/index.md [hunk 4]
```diff
@@ -118,7 +118,7 @@ function intersect(sphere, other) {
 
 Testing whether a sphere and an AABB are colliding is slightly more complicated, but still simple and fast. A logical approach would be to check every vertex of the AABB, doing a point vs sphere test for each one. This is overkill, however — testing all the vertices is unnecessary, as we can get away with just calculating the distance between the AABB's _closest point_ (not necessarily a vertex) and the sphere's center, seeing if it is less than or equal to the sphere's radius. We can get this value by clamping the sphere's center to the AABB's limits.
 
-![](sphere_vs_aabb.png)
+![Hand drawing of a square partially overlapping the top of a circle. The radius is denoted by a lite line labeled R. The  distance line goes from the center of the circle to the closest point of the square.](sphere_vs_aabb.png)
```
Comment (teoli2003) [inline files/en-us/games/techniques/3d_collision_detection/index.md:121]:
```suggestion
![Hand drawing of a square partially overlapping the top of a circle. The radius is denoted by a light line labeled R. The distance line goes from the circle's center to the closest point of the square.](sphere_vs_aabb.png)
```

File: files/en-us/games/techniques/3d_collision_detection/index.md [hunk 5]
```diff
@@ -95,7 +95,7 @@ function isPointInsideSphere(point, sphere) {
 
 The sphere vs sphere test is similar to the point vs sphere test. What we need to test here is that the distance between the sphere's centers is less than or equal to the sum of their radii.
 
-![](sphere_vs_sphere.png)
+![Hand drawing of two partially overlapping circles. Each circle as a light radius line going from the center of the circle to the border labeled R. The distance is denoted by a dotted line connecting the center points of both circles labeled D.](sphere_vs_sphere.png)
```
Comment (teoli2003) [inline files/en-us/games/techniques/3d_collision_detection/index.md:98]:
```suggestion
![Hand drawing of two partially overlapping circles. Each circle (of different sizes) has a light radius line going from its center to its border, labeled R. The distance is denoted by a dotted line, labeled D, connecting the center points of both circles.](sphere_vs_sphere.png)
```

File: files/en-us/web/performance/css_javascript_animation_performance/index.md [hunk 6]
```diff
@@ -40,10 +40,10 @@ In this section we'll walk you through a performance test, using Firefox, to see
 Before going through the example, please enable FPS tools first to see the current frame rate:
 
 1. In the URL bar, enter _about:config_; click the _I'll be careful, I promise!_ button to enter the config screen.
-    ![](pic1.png)
+    ![Warning screen that changing setting can be risky, with button to accept risks. ](pic1.png)
```
Comment (teoli2003) [inline files/en-us/web/performance/css_javascript_animation_performance/index.md:43]:
```suggestion
    ![Warning screen that changing settings can be risky, with a button to accept risks.](pic1.png)
```

File: files/en-us/web/performance/css_javascript_animation_performance/index.md [hunk 7]
```diff
@@ -40,10 +40,10 @@ In this section we'll walk you through a performance test, using Firefox, to see
...
 2. In the search bar, search for the `layers.acceleration.draw-fps` preference.
 3. Double-click the entry to set the value to `true`. Now you will be able to see three little purple boxes at the upper left corner of the Firefox window. The first box represents FPS.
-    ![](pic2.png)
+    ![Entering the search term filters the options. Only the draw-fps is showing, set to true, which led to the numbers 001, 001, and 108, appearing in the upper left corner of the browser, above the chrome. ](pic2.png)
```
Comment (teoli2003) [inline files/en-us/web/performance/css_javascript_animation_performance/index.md:46]:
```suggestion
    ![Entering the search term filters the options. Only the layers.acceleration.draw-fps preference is showing and is set to true. Three numbers (001, 001, and 108) are appearing in the upper left corner of the browser, overlaying its UI.](pic2.png)
```

File: files/en-us/web/performance/css_javascript_animation_performance/index.md [hunk 8]
```diff
@@ -169,7 +169,7 @@ Even given the test results above, we'd argue that CSS animations are the better
 
 To enable the OMTA (Off Main Thread Animation) in Firefox, you can go to _about:config_ and search for the `layers.offmainthreadcomposition.async-animations`. Toggle its value to `true`.
 
-![](pic3.png)
+![Entering the search term filters the options. Only the async animations is showing, set to true. The 3 numbers in the upper left corner of the browser, above the chrome, have increased to 005, 003, and 108. ](pic3.png)
```
Comment (teoli2003) [inline files/en-us/web/performance/css_javascript_animation_performance/index.md:172]:
```suggestion
![Entering the search term filters the options. Only the layers.offmainthreadcomposition.async-animations preference is showing, set to true. The three numbers in the upper left corner of the browser, above its UI, have increased to 005, 003, and 108.](pic3.png)
```

File: files/en-us/web/progressive_web_apps/installable_pwas/index.md [hunk 9]
```diff
@@ -87,27 +87,27 @@ A minimal web manifest must have at least a `name` and an `icons` field with at
 
 When the user visits the PWA with a supporting mobile browser, it should display a notification (such as a banner or dialog box) indicating that it's possible to install the app as a PWA.
 
-![Add to Home screen popup of js13kPWA.](js13kpwa-icon.png)
+![Popup explaining user can add site to home screen with link to continue to the web site.](js13kpwa-icon.png)
```
Comment (teoli2003) [inline files/en-us/web/progressive_web_apps/installable_pwas/index.md:90]:
```suggestion
![Screenshot of a mobile phone display with a popup saying "You can easily add this web site to your Home screen to have instant access and browse faster with an app-like experience", with a link to continue to the website below the text.](js13kpwa-icon.png)
```

File: files/en-us/web/progressive_web_apps/installable_pwas/index.md [hunk 10]
```diff
@@ -87,27 +87,27 @@ A minimal web manifest must have at least a `name` and an `icons` field with at
...
 
 After the user indicates they wish to proceed with installation, the install banner is shown. That banner is automatically created by the browser, based on the information from the manifest file. For instance, the prompt includes the app's name and icon.
 
-![Install banner of js13kPWA.](js13kpwa-banner.png)
+![Install banner with title of app, URL, and add to home screen button.](js13kpwa-banner.png)
```
Comment (teoli2003) [inline files/en-us/web/progressive_web_apps/installable_pwas/index.md:94]:
```suggestion
![Screenshot of a mobile phone display with an install banner at the top displaying the title of the app, its URL, and a screen-wide blue button reading "+ Add to home screen".](js13kpwa-banner.png)
```

File: files/en-us/web/progressive_web_apps/installable_pwas/index.md [hunk 11]
```diff
@@ -87,27 +87,27 @@ A minimal web manifest must have at least a `name` and an `icons` field with at
...
 
 If the user clicks the button, there is a final step showing what the app will look like, and letting the user choose if they definitely want to add the app.
 
-![Add to Home screen popup of js13kPWA.](js13kpwa-add.png)
+![Add to Home screen popup  with name of app, icon, and two buttons: cancel and add.](js13kpwa-add.png)
```
Comment (teoli2003) [inline files/en-us/web/progressive_web_apps/installable_pwas/index.md:98]:
```suggestion
![Screenshot of a mobile phone display with a modal window displaying "Add to Home screen" followed by the name of the app, its icon, and two buttons labeled "Cancel" and "Add".](js13kpwa-add.png)
```

File: files/en-us/web/progressive_web_apps/installable_pwas/index.md [hunk 12]
```diff
@@ -87,27 +87,27 @@ A minimal web manifest must have at least a `name` and an `icons` field with at
...
 
 When confirmed, the app will be installed on the home screen.
 
-![](js13kpwa-installed.png)
+![Screen shot showing the logo icon on the users phone home screen ](js13kpwa-installed.png)
```
Comment (teoli2003) [inline files/en-us/web/progressive_web_apps/installable_pwas/index.md:102]:
```suggestion
![Screenshot of the mobile phone Home screen displaying the logo icon of the application.](js13kpwa-installed.png)
```

File: files/en-us/web/progressive_web_apps/installable_pwas/index.md [hunk 13]
```diff
@@ -87,27 +87,27 @@ A minimal web manifest must have at least a `name` and an `icons` field with at
...
 
 Now the user can launch and use the web app just like any other application on their device. Depending on the device and operating system, the web app's icon may be badged with a small icon that indicates that it's a web app. In the screen shot above, for example, the app has a tiny Firefox icon, indicating that it's a web app that uses the Firefox runtime.
 
 ### Splash screen
 
 In some browsers, a splash screen is also generated from the information in the manifest, which is shown when the PWA is launched and while it's being loaded started up.
 
-![](js13kpwa-splash.png)
+![App splash screen: all red page with app logo in the middle](js13kpwa-splash.png)
```
Comment (teoli2003) [inline files/en-us/web/progressive_web_apps/installable_pwas/index.md:110]:
```suggestion
![Screenshot of the app's splash screen on a mobile phone. It is an all-red page with the application logo in the middle and its name below it: "js13kGames Progressive Web App"](js13kpwa-splash.png)
```

File: files/en-us/webassembly/using_the_javascript_api/index.md [hunk 14]
```diff
@@ -84,7 +84,7 @@ fetch('simple.wasm').then((response) =>
 
 In Firefox 54+, the Developer Tool Debugger Panel has functionality to expose the text representation of any wasm code included in a web page. To view it, you can go to the Debugger Panel and click on the "wasm://" entry.
 
-![](wasm-debug.png)
+![Deverloper tools debugger panel hiliting a module](wasm-debug.png)
```
Comment (teoli2003) [inline files/en-us/webassembly/using_the_javascript_api/index.md:87]:
```suggestion
![Developer tools debugger panel highlighting a module.](wasm-debug.png)
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]
👍 

This ALT-text project will make MDN so much better. I'm really happy!
